lib/short-css-class-names/index.js

```diff
@@ -12,33 +12,33 @@
 module.exports = function(rangeLength) {
     "use strict";
     
-    let charRange = require("./char-range"),
-        digits = charRange("0", "9"),
-        letters = charRange("A", "Z").concat(charRange("a", "z")),
-        hyphenAndUnderscore = ["-", "_"];
+    const charRange = require("./char-range");
+    const digits = charRange("0", "9");
+    const letters = charRange("A", "Z").concat(charRange("a", "z"));
+    const hyphenAndUnderscore = ["-", "_"];
 
-    let validFirstChars = letters,
-        allValidChars = digits.concat(letters).concat(hyphenAndUnderscore);
+    const validFirstChars = letters;
+    const allValidChars = digits.concat(letters).concat(hyphenAndUnderscore);
 
     let classNames = [];
     
     function tryAdd(className){
-        if(classNames.length >= rangeLength) return false;
+        if (classNames.length >= rangeLength) return false;
         classNames.push(className);
         return true;
     }
     
     validFirstChars.forEach(firstChar => {
         // Class names that are 1 char in length
-        if(!tryAdd(firstChar)) return;
+        if (!tryAdd(firstChar)) return;
```

Comment:
I prefer space before `(` when its not a function call which is also the convention of [aribnb](https://github.com/airbnb/javascript), which you referenced earlier. But you may change it as you please.